proposals/models.py

```diff
@@ -7,6 +7,10 @@
 from django.db import models
 from django.utils.translation import ugettext_lazy as _
 
+from django.utils.functional import lazy
+from django.utils.safestring import mark_safe
+mark_safe_lazy = lazy(mark_safe, str)
```

Comment:
Any reason we need to use the lazy mark_safe here?

Reply:
We used mark_safe_lazy with lazy translations elsewhere, so I was under under the impression that it was unwise to mix the reqular mark_safe with lazy translation. But I can't find specific references to this in the Django docs now that I'm looking for it.

We tend to use ugettext_lazy everywhere, I suppose for convenience, even when we don't strictly need it. So I thought this would be fine too. But marking safe has security implications so I'm fine with either.

Reply:
I tend to use mark_safe over mark_safe_lazy, mostly because I'm lazy and using mark_safe is easier. I suppose it doesn't really matter, I was just wondering. 

And year, ugettext_lazy is preferable imo as it almost always works, which can't be said for the non-lazy variant